### ISSUE TITLE
green glowstick not green

### ISSUE THREAD

**Issue by deltanedas** [space-wizards/space-station-14#16064]
## Description
saw on moose on miros

**Reproduction**
1. go dig top left of sci

**Screenshots**
![02:07:18](https://user-images.githubusercontent.com/39013340/236084289-96838509-2154-422e-8723-c429a365b657.png)

**Comment by themias** [comment]
I think the color just refers to the glow after you crack/activate them?
IIRC the color is clear/yellowish before activating it IRL.

**Comment by deltanedas** [comment]
yeah but im 90% sure it used to be green before lighting a while ago

**Comment by lzk228** [comment]
probably fixed

**Comment by lzk228** [comment]
as i remember, green glowstick is green
